### PR TITLE
test: test isUpdated functionality

### PR DESCRIPTION
**Pull request by james-a-morris**

### CODE REVIEW

**Comment by james-a-morris** [comment]
requires SDK push

File: test/BaseClass.isUpdated.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    const { spyLogger } = createSpyLogger();
+
+    const { hubPool, dai: l1Token } = await hubPoolFixture();
+    const { configStore } = await deployConfigStore(owner, [l1Token]);
+
+    const configStoreClient = new MockConfigStoreClient(spyLogger, configStore);
+    hubPoolClient = new MockHubPoolClient(spyLogger, hubPool, configStoreClient);
+
+    const { spokePool } = await deploySpokePool(ethers);
+    const deploymentBlock = await spokePool.provider.getBlockNumber();
```
Comment (pxrl) [inline test/BaseClass.isUpdated.ts:21]:
```suggestion
    const { spokePool, deploymentBlock } = await deploySpokePool(ethers);
```

File: test/BaseClass.isUpdated.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    it("should return false if the client is not updated", async () => {
+      expect(spokePoolClient.isUpdated).to.be.false;
+    });
+  });
+  describe("UBAClient", () => {
+    it("should return true if the client is updated", async () => {
+      await hubPoolClient.configStoreClient.update();
+      await hubPoolClient.update();
+      await spokePoolClient.update();
+      hubPoolClient.setLatestBlockNumber(50000);
```
Comment (pxrl) [inline test/BaseClass.isUpdated.ts:62]: OOC, what's the significance of this?

Comment (pxrl) [reply test/BaseClass.isUpdated.ts:62]: (Maybe just to satisfy that the searchConfig toBlock is set correctly?)

File: test/ConfigStoreClient.ts [hunk 3]
```diff
@@ -58,6 +58,14 @@ describe("AcrossConfigStoreClient", async function () {
   });
 
   it("update", async function () {
+    [owner] = await ethers.getSigners();
+    ({ dai: l1Token, weth: l2Token } = await hubPoolFixture());
+
+    configStore = await (await getContractFactory("AcrossConfigStore", owner)).deploy();
+    const { blockNumber: fromBlock } = await configStore.deployTransaction.wait();
```
Comment (pxrl) [inline test/ConfigStoreClient.ts:65]: Consider using `deployConfigStore()` here? It provides the deploymentBlock in the return object.

**Review by pxrl — APPROVED** [review]
Couple of points to consider, but otherwise LGTM.

File: test/BaseClass.isUpdated.ts (hunk 2 above)
Comment (james-a-morris) [reply test/BaseClass.isUpdated.ts:62]: Yes - this is to satisfy a block number

File: test/BaseClass.isUpdated.ts (hunk 1 above)
Comment (james-a-morris) [reply test/BaseClass.isUpdated.ts:21]: This currently isn't available as an export under `deploySpokePool`

File: test/ConfigStoreClient.ts (hunk 3 above)
Comment (james-a-morris) [reply test/ConfigStoreClient.ts:65]: This seems to actually break the tests - do you see a reason for the switch?

Comment (pxrl) [reply test/ConfigStoreClient.ts:65]: Interesting; that's a surprise. But no issue with the way it's implemented here :+1:

**Pull request merged**